README.md

```diff
@@ -182,6 +182,15 @@ kubectl create -f deploy/demo/deployment.yaml
 
 > There's also a detailed tutorial [here](docs/tutorial/README.md).
 
+
+# Uninstall notes
+Upon deploying pod identity (NMI) modifies nodes' iptables to allow it to intercept calls to Azure Instance Metadata endpoint. This allows NMI to assert identities assigned to pod before executing the request on behalf of the caller. When you uninstall pod identity these iptables entries will remain. You can remove them with
+
+```
+iptables -t nat -F
```

Comment:
This will flush out all the entries. The manual steps for removing just the aad-metadata should be - 

1. iptables -t nat -D PREROUTING -j aad-metadata
2. iptables -t nat -F aad-metadata
3. iptables -t nat -X aad-metadata

This will remove the reference links, flush only the `aad-metadata` chain and then delete it.

Reply:
@khenidak I've a PR up to do the clean up as part of container termination - https://github.com/Azure/aad-pod-identity/pull/211